types/model.d.ts

```diff
@@ -328,6 +520,24 @@ export declare class ModelBase {
    * @end
    */
   static clearObservers(operation: string): void;
+
+  getMergePolicy(options: {
+    configureModelMerge: boolean | object
+  }): {
+    description?: {replace: boolean};
+    properties?: {patch: boolean}; // Only referenced in "legacy built-in merge policy"
+    options?: {path: boolean};
+    hidden?: {replace: boolean};
+    protected?: {replace: boolean};
+    indexes?: {patch: boolean};
+    methods?: {patch: boolean};
+    mixins?: {patch: boolean};
+    relations?: {patch: boolean};
+    scope?: {replace: boolean};
+    acls?: {rank: boolean};
+    __delete?: boolean | null;
+    __default?: {replace: boolean};
+  }
 }
```

Comment:
Spin off model merge config to its own interface; Add tsdoc for each config property.